--- SPIRV-Tools-5834719fc17d4735fce0102738b87b70255cfd5f/source/CMakeLists.txt
+++ SPIRV-Tools-5834719fc17d4735fce0102738b87b70255cfd5f-new/source/CMakeLists.txt
@@ -287,7 +287,7 @@
   ${CMAKE_CURRENT_SOURCE_DIR}/software_version.cpp
   PROPERTIES OBJECT_DEPENDS "${SPIRV_TOOLS_BUILD_VERSION_INC}")
 
-add_library(${SPIRV_TOOLS} ${SPIRV_SOURCES})
+add_library(${SPIRV_TOOLS} STATIC ${SPIRV_SOURCES})
 spvtools_default_compile_options(${SPIRV_TOOLS})
 target_include_directories(${SPIRV_TOOLS}
   PUBLIC ${spirv-tools_SOURCE_DIR}/include

--- SPIRV-Tools-5834719fc17d4735fce0102738b87b70255cfd5f/source/comp/CMakeLists.txt
+++ SPIRV-Tools-5834719fc17d4735fce0102738b87b70255cfd5f-new/source/comp/CMakeLists.txt
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 if(SPIRV_BUILD_COMPRESSION)
-  add_library(SPIRV-Tools-comp markv_codec.cpp)
+  add_library(SPIRV-Tools-comp STATIC markv_codec.cpp)
 
   spvtools_default_compile_options(SPIRV-Tools-comp)
   target_include_directories(SPIRV-Tools-comp

--- SPIRV-Tools-5834719fc17d4735fce0102738b87b70255cfd5f/source/opt/CMakeLists.txt
+++ SPIRV-Tools-5834719fc17d4735fce0102738b87b70255cfd5f-new/source/opt/CMakeLists.txt
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-add_library(SPIRV-Tools-opt
+add_library(SPIRV-Tools-opt STATIC
   aggressive_dead_code_elim_pass.h
   basic_block.h
   block_merge_pass.h
--- SPIRV-Tools-5834719fc17d4735fce0102738b87b70255cfd5f/source/link/CMakeLists.txt
+++ SPIRV-Tools-5834719fc17d4735fce0102738b87b70255cfd5f-new/source/link/CMakeLists.txt
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-add_library(SPIRV-Tools-link
+add_library(SPIRV-Tools-link STATIC
   linker.cpp
 )
 

